src/HonzaBotner/appsettings.Development.json

```diff
@@ -8,28 +8,28 @@
         }
     },
     "Discord": {
-        "GuildId": 887815108720467998,
-        "LogChannelId": 968247366849282062
+        "GuildId": 1043604803302981702,
+        "LogChannelId": 1043604805186240532
     },
     "CVUT": {
         "AppBaseUrl": "https://localhost:5001"
     },
     "CommonCommandOptions": {
-        "ModRoleId": 887978558759264296,
-        "BotRoleId": 887978852930953246,
-        "AuthenticatedRoleId": 887977978712174622,
+        "ModRoleId": 1043604803625943088,
+        "BotRoleId": 1043604803613376566,
+        "AuthenticatedRoleId": 1043604803604983844,
         "HugEmoteName": "",
-        "BoosterRoleId": 820565262095876158,
-        "GentlemenChannelId": 820299208120336415,
+        "BoosterRoleId": 0,
+        "GentlemenChannelId": 0,
         "GentlemenFilePath": "Static/gentleman.gif",
-        "HornyJailRoleId": 760050336366329856,
-        "HornyJailChannelId": 819986928928686080,
+        "HornyJailRoleId": 1043604803584016549,
+        "HornyJailChannelId": 1043604815701360714,
         "HornyJailFilePath": "Static/hornyjail.mp4",
-        "StandUpRoleId": 972123386094440498,
-        "StandUpChannelId": 750108543125946448
+        "StandUpRoleId": 1044387139154100355,
+        "StandUpChannelId": 1043604815701360713
     },
     "CustomVoiceOptions": {
-        "ClickChannelId": 887815109165076500,
+        "ClickChannelId": 1043604816154341398,
         "RemoveAfterCommandInSeconds": 30
```

Comment:
Are you sure about this?

Reply:
That was actually missclick from my side, although the existing values are from my testing server anyways :sweat_smile: